code/modules/mob/living/brain/posibrain.dm

```diff
@@ -88,7 +88,13 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 /obj/item/mmi/posibrain/proc/activate(mob/user)
 	if(QDELETED(brainmob))
 		return
-	if(is_occupied() || is_banned_from(user.ckey, ROLE_POSIBRAIN) || QDELETED(brainmob) || QDELETED(src) || QDELETED(user))
+	if(is_banned_from(user.ckey, ROLE_POSIBRAIN))
+		to_chat(user, "<span class='warning'>You are restricted from taking positronic brain spawns at this time.</span>")
+		return
+	if(user.client.get_exp_living(TRUE) <= 120)
```

Comment:
Fixing this would mean making defines for every hours played requirement.

Tbh not a bad idea but seems somewhat out of scope for this. Do you want me to go ahead and make it part of this?

Reply:
There you go, set defines for all hours played requirements on jobs